gremlin-python/src/main/python/setup.py

```diff
@@ -75,7 +75,7 @@
     tests_require=[
         'pytest>=4.6.4,<7.2.0',
         'mock>=3.0.5,<4.0.0',
-        'radish-bdd==0.8.6',
+        'radish-bdd==0.13.4',
```

Comment:
Just to note that I've bumped the radish test suite to the latest version. 

While it no longer matters inside of Docker (tests pass for either version), it resolves the error `Setup script exited with error in radish-parse_type setup command: use_2to3 is invalid.` for the previous pom set up.